force-app/main/default/classes/UpdateContactPhoneNumber.cls

```diff
@@ -17,15 +17,21 @@ public class UpdateContactPhoneNumber implements Database.Batchable<sObject>, Da
 
     public void execute(Database.BatchableContext bc, List<Contact> scope){
 
-        List<Contact> contactsToBeUpdated = new List<Contact>();
-
-        for(Contact contactRecord : scope){
-            contactRecord.Phone = phoneToBeUpdated;
-            recordsProcessed++;
-            contactsToBeUpdated.Add(contactRecord);
-        }    
-
-        update contactsToBeUpdated;
+        try{
+            List<Contact> contactsToBeUpdated = new List<Contact>();
+
+            for(Contact contactRecord : scope){
+                contactRecord.Phone = phoneToBeUpdated;
+                recordsProcessed++;
+                contactsToBeUpdated.Add(contactRecord);
+            }    
+    
+            update contactsToBeUpdated;
+        }
+        catch(Exception ex){
+			HandleException Handle = new HandleException(ex);
```

Comment:
Were you able to call the HandleException constructor only with one parameter Exception? 
I see that the constructor has 2 parameters:
 public HandleException(Exception ex, String ClassName){